1.1/reqs.ttl

```diff
@@ -0,0 +1,951 @@
+@prefix confs10: <http://www.opengis.net/spec/geosparql/1.0/conf/> .
+@prefix confs11: <http://www.opengis.net/spec/geosparql/1.1/conf/> .
+@prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix conf10gx: <http://www.opengis.net/spec/geosparql/1.0/conf/geometry-extension/> .
+@prefix conf11gx: <http://www.opengis.net/spec/geosparql/1.1/conf/geometry-extension/> .
+@prefix reqs10gx: <http://www.opengis.net/spec/geosparql/1.0/req/geometry-extension/> .
+@prefix reqs11gx: <http://www.opengis.net/spec/geosparql/1.1/req/geometry-extension/> .
+@prefix conf10gtx: <http://www.opengis.net/spec/geosparql/1.0/conf/geometry-topology-extension/> .
+@prefix conf10qre: <http://www.opengis.net/spec/geosparql/1.0/conf/query-rewrite-extension/> .
+@prefix conf10ree: <http://www.opengis.net/spec/geosparql/1.0/conf/rdfs-entailment-extension/> .
+@prefix conf10tve: <http://www.opengis.net/spec/geosparql/1.0/conf/topology-vocab-extension/> .
+@prefix conf11gtx: <http://www.opengis.net/spec/geosparql/1.1/conf/geometry-topology-extension/> .
+@prefix conf11qre: <http://www.opengis.net/spec/geosparql/1.1/conf/query-rewrite-extension/> .
+@prefix conf11ree: <http://www.opengis.net/spec/geosparql/1.1/conf/rdfs-entailment-extension/> .
+@prefix conf11tve: <http://www.opengis.net/spec/geosparql/1.1/conf/topology-vocab-extension/> .
+@prefix reqs10gtx: <http://www.opengis.net/spec/geosparql/1.0/req/geometry-topology-extension/> .
+@prefix reqs10qre: <http://www.opengis.net/spec/geosparql/1.0/req/query-rewrite-extension/> .
+@prefix reqs10ree: <http://www.opengis.net/spec/geosparql/1.0/req/rdfs-entailment-extension/> .
+@prefix reqs10tve: <http://www.opengis.net/spec/geosparql/1.0/req/topology-vocab-extension/> .
+@prefix reqs11gtx: <http://www.opengis.net/spec/geosparql/1.1/req/geometry-topology-extension/> .
+@prefix reqs11qre: <http://www.opengis.net/spec/geosparql/1.1/req/query-rewrite-extension/> .
+@prefix reqs11ree: <http://www.opengis.net/spec/geosparql/1.1/req/rdfs-entailment-extension/> .
+@prefix reqs11tve: <http://www.opengis.net/spec/geosparql/1.1/req/topology-vocab-extension/> .
+@prefix conf10core: <http://www.opengis.net/spec/geosparql/1.0/conf/core/> .
+@prefix conf11core: <http://www.opengis.net/spec/geosparql/1.1/conf/core/> .
+@prefix reqs10core: <http://www.opengis.net/spec/geosparql/1.0/req/core/> .
+@prefix reqs11core: <http://www.opengis.net/spec/geosparql/1.1/req/core/> .
+
+@base <http://www.opengis.net/def/geosparql/reqs> .
+<http://www.opengis.net/def/geosparql/reqs> rdf:type owl:Ontology ;
+                                             dcterms:created "2021-06-28"^^xsd:date ;
+                                             dcterms:modified "2021-07-13"^^xsd:date ;
+                                             rdfs:seeAlso <http://www.opengis.net/doc/geosparql/1.1> ;
+                                             skos:definition "A vocabulary of the requirements and conformance classes in the GeoSPARQL 1.1 specification"@en ;

```

Comment:
Change to "... requirements classes and conformance classes ..."?
And shouldn't sentences like these have proper punctuation? That is something we need to agree on for all RDF code. I have just extended [the project wiki](https://github.com/opengeospatial/ogc-geosparql/wiki) to be able to set text formatting guidelines.

Reply:
I guess it is a matter of discussion. I somehow had in mind that skos definitions were not supposed to end with a dot

Reply:
I tend not to use training full stop but often have structured literals for which I need to include internal punctuation e.g.:

```turtle
"""This is a sentence.

This is a second.

No training full stop here"""@en
```

Reply:
What is the reason for omitting the full stop?